resources/machines/ultimaker2_extended_plus_025.json

```diff
@@ -7,7 +7,8 @@
     "platform": "ultimaker2_platform.obj",
     "platform_texture": "ultimaker2plus_backplate.png",
     "inherits": "ultimaker2_extended_plus.json",
-    "variant": "0.25mm Nozzle",
+    "variant": "0.25 mm",
```

Comment:
In the UI, the field is labeled "Nozzle" now. So the full UI would read:
"Nozzle: 0.25mm Nozzle"